js_modules/dagit/packages/core/src/ticks/EvaluateSchedule.tsx

```diff
@@ -0,0 +1,400 @@
+import {gql, useMutation, useQuery} from '@apollo/client';
+import {
+  Box,
+  Button,
+  Code,
+  Colors,
+  Dialog,
+  DialogBody,
+  DialogFooter,
+  Icon,
+  Menu,
+  MenuItem,
+  Mono,
+  NonIdealState,
+  Popover,
+  Spinner,
+  Subheading,
+  Tag,
+} from '@dagster-io/ui';
+import React from 'react';
+import styled from 'styled-components/macro';
+
+import {PYTHON_ERROR_FRAGMENT} from '../app/PythonErrorFragment';
+import {PythonErrorInfo} from '../app/PythonErrorInfo';
+import {TimezoneContext} from '../app/time/TimezoneContext';
+import {timestampToString} from '../app/time/timestampToString';
+import {useViewport} from '../gantt/useViewport';
+import {testId} from '../testing/testId';
+import {repoAddressToSelector} from '../workspace/repoAddressToSelector';
+import {RepoAddress} from '../workspace/types';
+
+import {RunRequestTable} from './DryRunRequestTable';
+import {
+  GetScheduleQuery,
+  GetScheduleQueryVariables,
+  ScheduleDryRunMutation,
+  ScheduleDryRunMutationVariables,
+} from './types/EvaluateSchedule.types';
+
+const locale = navigator.language;
+
+export const EvaluateSchedule: React.FC<{
+  repoAddress: RepoAddress;
+  name: string;
+  onClose: () => void;
+  isOpen: boolean;
+  jobName: string;
+}> = ({repoAddress, name, isOpen, onClose, jobName}) => {
+  const [_selectedTimestamp, setSelectedTimestamp] = React.useState<{ts: number; label: string}>();
+  const {data} = useQuery<GetScheduleQuery, GetScheduleQueryVariables>(GET_SCHEDULE_QUERY, {
+    variables: {
+      scheduleSelector: {
+        repositoryLocationName: repoAddress.location,
+        repositoryName: repoAddress.name,
+        scheduleName: name,
+      },
+    },
+  });
```

Comment:
Hmm this is a query within a dialog, but it's not lazy or using `skip: !isOpen` so I believe this is being called for every schedule row even if you're not opening the dialogs? 

To make this simpler (and to avoid the key={1 | 0} thing above), it might be nice to just render this content when the modal is open and sacrifice the brief animation. This is also my pitch for the imperative style `onClick={() => showCustomAlert(<EvaluateScheduleModal />)}` but I don't think we do that except for alerts.